### DISCUSSION TITLE
find code very difficult

### DISCUSSION THREAD

**Discussion by kakashianh** [community/community#54687] [New to GitHub]
i find code sort by recently as github not update very easy, but now github return result as shit

**Comment by queenofcorgis** [comment]
Hi @githubanhduong , we're happy you're here! You are more likely to get a useful response if you are explicit about what your question entails, giving a few more details might help someone give you a nudge in the right direction.

**Reply by kakashianh** [reply]
![image](https://github.com/community/community/assets/87347412/5fc1c7bb-503e-45aa-8d67-b005ab9dbd6c)

example i want find code sort by recently update , but github have miss it

**Reply by kakashianh** [reply]
find code just writed will help me find corresponding project

**Reply by kakashianh** [reply]
for me skype or facebook to me  connect you
